src/components/PathNav.tsx

```diff
@@ -0,0 +1,57 @@
+"use client";
+
+import React from "react";
+import { usePathname } from "next/navigation";
+
+interface PathNavParams {
+  pathInfo: PathInfoType[];
+}
+
+export type PathInfoType = {
+  display: string;
+  url: string;
+};
+
+const PathNav = ({ pathInfo }: PathNavParams) => {
+  const currentPath = "http://localhost:3000" + usePathname();
```

Comment:
We don't need `http://localhost:3000` here! Navigation within the application can be relative to `/`. So `usePathName()` by itself will achieve the same functionality